.github/workflows/trigger-dependent-repo-update.yml

```
@@ -0,0 +1,26 @@
+name: Trigger Dependent Repo Update Workflow
```

```suggestion
name: Trigger dependent repo update workflow
```

Other workflows don't seem to be using title case.